libs/community/langchain_community/utilities/vertexai.py

```diff
@@ -80,6 +94,9 @@ def init_vertexai(
         credentials=credentials,
     )
 
+    _, user_agent = _get_user_agent(module)
+    initializer.global_config.append_user_agent(user_agent)
```

Comment:
getting mypy errors that this `append_user_agent` function doesn't exist on `_Config`. Also causes the tests to fail, so seems like a real problem even after updating packages.

@holtskinner @lkuligin let me know how's best to fix!

Reply:
Oh I'm silly - looks like it'll be in 42 when ready. Will convert this to a draft PR for now!